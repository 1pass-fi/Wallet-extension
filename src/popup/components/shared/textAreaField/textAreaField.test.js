import '@babel/polyfill'
import { fireEvent, render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'

import TextAreaField from '.'

describe('Test for TextAreaField component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<TextAreaField />)
      expect(container).toMatchSnapshot()
      const toggleDisplayElement = document.querySelector('.toggle-display')
      fireEvent.click(toggleDisplayElement)
      expect(container).toMatchSnapshot()
    })
  })
})
