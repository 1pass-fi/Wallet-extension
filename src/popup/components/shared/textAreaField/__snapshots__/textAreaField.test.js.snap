// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for TextAreaField component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="text-area-field "
  >
    <div
      class="label"
    />
    <div
      class="input-wrapper"
    >
      <textarea
        class="input protect"
        name=""
        placeholder=""
        spellcheck="false"
      />
      <svg
        class="toggle-display"
      />
    </div>
  </div>
</div>
`;

exports[`Test for TextAreaField component Render without crashing renders correctly 2`] = `
<div>
  <div
    class="text-area-field "
  >
    <div
      class="label"
    />
    <div
      class="input-wrapper"
    >
      <textarea
        class="input true"
        name=""
        placeholder=""
        spellcheck="false"
      />
      <svg
        class="toggle-display"
      />
    </div>
  </div>
</div>
`;
